stellarphot/settings/models.py

```diff
@@ -499,10 +545,28 @@ class SourceLocationSettings(BaseModelWithTableRep):
     """
 
     source_list_file: Annotated[
-        str, Field(json_schema_extra=dict(autoui="ipyautoui.custom.FileChooser"))
+        str,
+        Field(
+            json_schema_extra=dict(
+                autoui="ipyautoui.custom.FileChooser",
+                filter_pattern=["*.ecsv", "*.csv"],
+            )
+        ),
     ]
-    use_coordinates: Literal["sky", "pixel"] = "sky"
-    shift_tolerance: NonNegativeFloat = 5.0
+    use_coordinates: Annotated[
+        Literal["sky", "pixel"],
+        Field(description="coordinates to use for locating sources"),
```

Comment:
Do we need to add "in images" after "locating sources"?  Probably just me being overly specific.

Reply:
Yes, I think that is helpful